test/python/test_nixl_api.py

```
@@ -127,6 +127,35 @@ def test_nixl_conf_bad_sync_mode():
+@pytest.mark.parametrize(
+    ("configured_mode", "expected_mode"),
+    [
+        ("none", "none"),
+        ("peer", "peer"),
+        (None, None),
+    ],
+)
+def test_ucx_error_handling_mode_config(monkeypatch, configured_mode, expected_mode):
+    captured = {}
+
+    def spy_create_backend(self, backend, init_params):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `spy_create_backend`

Add return type annotation: `None`

(ANN202)